contracts/cw20-atomic-swap/src/balance.rs

```diff
@@ -1,27 +1,34 @@
 use schemars::JsonSchema;
 use serde::{Deserialize, Serialize};
 
-use cosmwasm_std::Coin;
 use cw20::Cw20Coin;
 
 #[derive(Serialize, Deserialize, Clone, Debug, PartialEq, JsonSchema)]
 #[serde(rename_all = "snake_case")]
 pub enum Balance {
```

Comment:
Balance and `cw0::Balance` is a bit confusing.

Do you have an idea for a different name for one of them? `cw0::NativeBalance`?

Reply:
Mi idea was that the `cw0` of `cw0::Balance` implies native. But I agree it's confusing. It could be `NativeBalance` or `AtomBalance`. As you like / prefer.

Reply:
Refactored in favor of `NativeBalance`.